src/dbt_score/rule.py

```diff
@@ -73,7 +82,7 @@ def process_config(self, rule_config: RuleConfig) -> None:
 
         # Overwrite default rule configuration
         for k, v in rule_config.config.items():
-            if k in self.default_config:
+            if k in self.default_config or k == "model_filter_names":
```

Comment:
This `model_filter_names` can be implemented in the `RuleConfig` class. It should be treated similarly to `Severity`

Reply:
Then it can be added to `Rule` later on in `process_config`, so it becomes a Rule property